p2p/ipld/read_test.go

```diff
@@ -111,16 +111,7 @@ func TestGetLeafData(t *testing.T) {
 	data := generateRandNamespacedRawData(16, types.NamespaceSize, types.ShareSize)
 
 	// create a random tree
-	tree, err := createNmtTree(ctx, batch, data)
-	if err != nil {
-		t.Error(err)
-	}
-
-	// calculate the root
-	root := tree.Root()
-
-	// commit the data to IPFS
-	err = batch.Commit()
+	root, err := createNmtTree(ctx, batch, data)
```

Comment:
I feel like the method should be renamed now. Of course, it still creates the tree but it also returns the root now. Maybe call it `getTreeRoot`?

Reply:
Nice, I was thinking about that and was distracted by something and simply forgot. Thanks!